inst/stan/epinowcast.stan

```diff
@@ -392,25 +394,44 @@ model {
   if (likelihood) {
     profile("model_likelihood") {
     if (ll_aggregation) {
-      target += reduce_sum(
-        delay_group_lupmf, groups, 1, flat_obs, lsl, clsl, nsl, cnsl,
-        flat_obs_lookup, exp_lobs, t, sg, ts, st, rep_findex, srdlh, refp_lh,
-        refp_findex, model_refp, rep_fncol, ref_as_p, phi, model_obs, model_miss, miss_obs, missing_reference,
-        obs_by_report, miss_ref_lprop, sdmax, csdmax, miss_st, miss_cst,
-        refnp_lh, model_refnp
-      );
+      if (parallelise_likelihood) {
+        target += reduce_sum(
+          delay_group_lupmf, groups, 1, flat_obs, lsl, clsl, nsl, cnsl,
+          flat_obs_lookup, exp_lobs, t, sg, ts, st, rep_findex, srdlh, refp_lh,
+          refp_findex, model_refp, rep_fncol, ref_as_p, phi, model_obs, model_miss, miss_obs, missing_reference,
+          obs_by_report, miss_ref_lprop, sdmax, csdmax, miss_st, miss_cst,
+          refnp_lh, model_refnp
+        );
+      } else {
+        target += delay_group_lupmf(
```

Comment:
I think we can simplify this and only use `delay_group_lupmf` if the likelihood isn't in parallel and then other the two options (group or snapshot if it is) the only difference between these two is how they parallelise with the group option being the only one that can be done in parallel with support for missing data imputation (and so this should be the none parallel option).

Reply:
we could also potentialyl use the none target based syntax for the non-parallel version (i.e obs ~ delay,....).